src/core/components/AuthenticateButton/index.js

```diff
@@ -53,9 +55,17 @@ export const mapStateToProps = (state) => ({
 });
 
 export const mapDispatchToProps = (dispatch) => ({
-  handleLogOut() {
+  handleLogOut({ api }) {
+    const { location } = window;
+    if (config.get('apiHost') === `${location.protocol}//${location.host}`) {
+      // Deployed config, server manages cookie.
+      return logOutFromServer({ api })
+        .then(() => dispatch(logOutUser()));
```

Comment:
Maybe the error can be flagged as "global" and we can put a display only component in `App` that renders global errors. I guess clearing these could be tricky but we could clear all global errors on navigation or put a close button on the error.

```jsx
// AuthenticateButton
return logOutFromServer({ api })
  .then(() => success())
  .catch((error) => dispatch(addError({
    error, name: 'AuthenticateButton', global: true,
   })));

// App
<div className="content">
  <ErrorHandler global />
  {/* ... */}
</div>